viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -32,30 +27,24 @@ struct ColorARGB {
     unsigned char B;
 };
 
-// Check if pixel is the default color signaling no data is present
-bool CheckIfEmptyPixel(ColorARGB pixel_color) {
-    return (pixel_color.R == defaultNoDataRGBValue) &&
-           (pixel_color.G == defaultNoDataRGBValue) &&
-           (pixel_color.B == defaultNoDataRGBValue);
-}
+// Check if the green color channel is 0 to filter unobserved pixels which is set in 
+// DrawTexture at https://github.com/cartographer-project/cartographer/blob/master/
```

Comment:
[nit] Could we use a permalink like https://github.com/cartographer-project/cartographer/blob/ef00de2317dcf7895b09f18cc4d87f8b533a019b/cartographer/io/submap_painter.cc#L206-L207 in case anything in that file changes?